docs/api/cli-options.md

```diff
@@ -45,9 +47,15 @@ Options include:
 | `--ci`                          | CI mode (skip interactive prompts, don't open browser)<br/>`storybook dev --ci`                                                                                           |
 | `--no-open`                     | Do not open Storybook automatically in the browser<br/>`storybook dev --no-open`                                                                                          |
 | `--quiet`                       | Suppress verbose build output<br/>`storybook dev --quiet`                                                                                                                 |
+| `--debug`                       | Outputs more logs in the CLI to assist debugging<br/>`storybook dev --debug`                                                                                              |
 | `--debug-webpack`               | Display final webpack configurations for debugging purposes<br/>`storybook dev --debug-webpack`                                                                           |
-| `--stats-json`          | Write  stats JSON to disk<br/>`storybook dev ---stats-json /tmp/stats`<br/>NOTE: only works for webpack.                                                                              |
+| `--stats-json [dir-name]`       | Write stats JSON to disk<br/>`storybook dev --stats-json /tmp/stats`<br/>NOTE: only works for Webpack.                                                                    |
+| `--no-version-updates`          | Suppress Storybook's update check<br/>`storybook dev --no-version-updates`                                                                                                |
 | `--docs`                        | Starts Storybook in documentation mode. Learn more about it in [here](../writing-docs/build-documentation.md#preview-storybooks-documentation)<br/>`storybook dev --docs` |
+| `--initial-path [path]`         | URL path to be appended when visiting Storybook for the first time<br/>`storybook dev --initial-path=/docs/getting-started--docs`                                         |
```

Comment:
Applied, but with a slight modification:
```
Configures the URL Storybook should open when it opens the browser for the first time
```

because that's only applicable when the browser is auto-opened by Storybook